binderhub/tests/conftest.py

```diff
@@ -221,7 +219,8 @@ def always_build(app, request):
     """
```

Comment:
This is pretty clever, now that I better understand what it does/how to use it :)

Reply:
Normally I dislike test related stuff that makes me go "aha, this is clever", but yeah. I might make a PR later that doesn't patch `get_build_slug` and somehow has a very long slug.